src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -1,8 +1,18 @@
 package core.basesyntax;
 
-/**
- * Feel free to remove this class and create your own.
- */
 public class HelloWorld {
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        ColorSupplier colorSupplier = new ColorSupplier();
+        Figure[] figures = new Figure[6];
```

Comment:
All magic numbers in your code should be constants.
Please see [this article](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names) to learn about constant fields and their naming requirements.